core/mm/core_mmu.c

```diff
@@ -2552,6 +2602,29 @@ static TEE_Result teecore_init_pub_ram(void)
 early_init(teecore_init_pub_ram);
 #endif /*CFG_CORE_RESERVED_SHM*/
 
+static bool get_extra_ta_ram(vaddr_t *s, vaddr_t *e)
+{
+	struct tee_mmap_region *map = NULL;
+	struct tee_mmap_region *lower_ta_ram = NULL;
+
+	for (map = get_memory_map(); !core_mmap_is_end_of_table(map); map++) {
+		if (map->type == MEM_AREA_TA_RAM) {
+			DMSG("map = { .va=%lx, .size=%lx }", map->va, map->size);
```

Comment:
`			DMSG("map = { .va=%#"PRIxVA", .size=%#zx }", map->va, map->size);`

Reply:
Oh this was used while debugging and I think this can be removed form the commit